tests/deployment/ipmi.py

```diff
@@ -86,6 +86,9 @@ def test_node_set_bootdev(self):
                 api.node_set_bootdev(node.label, 'invalid-device')
 
         # register a node with erroneous ipmi details to raise OBMError
+        # XXX: In theory, this could actually be a real node; we should take
+        # some measure to ensure this never collides with something actually
```

Comment:
(We don't even have to haveit be random every time -- just do it once to make sure the user isn't going to cause a collision by accident).